Algorithms/Sorting/Tree_Sort.py

```diff
@@ -0,0 +1,84 @@
+''' SORTING USING BINARY SEARCH TREE
+    Binary Search Tree is a special type of binary tree where
+    1. The value of all the nodes in the left sub-tree is less than or equal to the value of the root.
+    2. The value of all the nodes in the right sub-tree is greater than value of the root.
+    3. This rule will be recursively applied to all the left and right sub-trees of the root.
+
+    Inorder traversal
+    1. Traverse the left subtree
+    2. Visit the root.
+    3. Traverse the right subtree
+
+    Inorder traversal in BST gives sorted list
```

Comment:
Explain how the tree is printed during In order Traversal

Reply:
It is explained in lines 8-10